components/style/properties/longhand/font.mako.rs

```diff
@@ -1792,10 +1792,15 @@ ${helpers.single_keyword_system("font-variant-position",
 
         impl ToCss for FeatureTagValue {
             fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+                use std::{mem,str};
+
+                let raw: [u8;4] = unsafe{mem::transmute(self.tag)};
```

Comment:
Please don't use unsafe code for stuff like this. Either reconstruct it from the mask, or use the byteorder crate to get a safe version of this transmute.